HPXMLtoOpenStudio/tests/test_hvac.rb

```diff
@@ -1104,7 +1104,7 @@ def test_heat_pump_advanced_defrost
 
     # Ductless heat pump test
     args_hash = {}
-    args_hash['hpxml_path'] = File.absolute_path(File.join(@sample_files_path, 'base-hvac-mini-split-heat-pump-ductless-advanced-defrost.xml'))
+    args_hash['hpxml_path'] = File.absolute_path(File.join(@sample_files_path, 'base-hvac-mini-split-heat-pump-ductless-backup-advanced-defrost.xml'))
```

Comment:
Updated the test to have integrated backup to demonstrate that we still don't model backup heat during defrost by default (because it's ductless).